Makefile-libostree.am

```diff
@@ -260,6 +260,20 @@ libostree_1_la_CFLAGS += $(OT_DEP_SELINUX_CFLAGS)
 libostree_1_la_LIBADD += $(OT_DEP_SELINUX_LIBS)
 endif
 
+libostree_1_la_SOURCES += \
```

Comment:
Is this supposed to be build `--enable-experimental-api=no` ?

Reply:
Yes